src/components/ClickableMap/ClickableMap.js

```diff
@@ -95,18 +100,23 @@ function ClickableMap({
       // Find city and state.
       let locality = null;
       let administrativeAreaLevel1 = null;
+      let administrativeAreaLevel3 = null;
       result.address_components.forEach((addressComp) => {
         if (addressComp.types.indexOf('locality') !== -1) {
           locality = addressComp.long_name;
         }
         if (addressComp.types.indexOf('administrative_area_level_1') !== -1) {
           administrativeAreaLevel1 = addressComp.short_name;
         }
+        if (addressComp.types.indexOf('administrative_area_level_3') !== -1) {
```

Comment:
No need to change, but just calling out - if you are just checking for existing of one string in another you can use `includes` (i.e. `!addressComp.types.includes('administrative_area_level_3')`

Reply:
My bad habit! :) I have had to worry about IE11 support in the past on other projects and it was easy to use indexOf without a polyfill so I have been doing that.